runtimes/eden/src/lib.rs

```diff
@@ -381,6 +384,65 @@ sp_api::impl_runtime_apis! {
 
 			impl frame_system_benchmarking::Config for Runtime {}
 
+			use xcm::latest::prelude::*;
+			use frame_benchmarking::BenchmarkError;
+			use pallet_xcm_benchmarks::generic::Pallet as XcmBenchmarks;
+			impl pallet_xcm_benchmarks::Config for Runtime {
```

Comment:
Can you please move this to `xcm_config.rs`?

Reply:
Done